test.py

```diff
@@ -0,0 +1,69 @@
+import unittest
+from dojo import Dojo
+
+class TestCreateRoom(unittest.TestCase):
+    """Test cases for creating room"""
+    def setUp(self):
+        self.dojo = Dojo()
+
+    def test_adds_office_successfully(self):
+        """Tests if room of type office can be added successfully"""
+        office = self.dojo.create_room("office", "A04")
+        self.assertEqual(office, "Office added successfully")
+
+    def test_adds_living_space_successfully(self):
+        """Tests if room of type livingSpace can be added successfully"""
+        living_space =self.dojo.create_room("living_space", "Arusha")
+        self.assertEqual(living_space, "Living space added successfully")
+
+    def test_room_exists(self):
+        """Tests if room already exists"""
+        self.dojo.create_room("office", "Bujumbura")
+        room =self.dojo.create_room("office", "Bujumbura")
+        self.assertEqual(room,"Room already exists")
+
+    def test_create_wrong_room_type(self):
+        """Tests adding wrong room type """
+        room = self.dojo.create_room("social_hall", "Stam")
+        self.assertEqual(room, "Wrong room type")
+
+class TestAddingPersons(unittest.TestCase):
+    """Test cases for Adding Persons"""
+    def setUp(self):
+        self.dojo = Dojo()
+        self.dojo.fellows = []
```

Comment:
this list should not be initialised in the test setUp, you've created an instance of the class and this will include the list you declared in the class Dojo.